gui/wxpython/modules/import_export.py

```diff
@@ -527,7 +527,7 @@ def OnRun(self, event):
             if nBandsStr:
                 try:
                     nBands = int(nBandsStr.rstrip("\n"))
-                except:
+                except (ValueError, TypeError):
```

Comment:
How would you get TypeError here?

Reply:
Incase objectType is not string. I have also added `AttributeError` since i think it should be caught by `rstrip` itself

Reply:
`nBandsStr` will be a string (maybe None, but that is covered by the if above).